test/release/examples/primers/chplvis/prog1.chpl

```diff
@@ -0,0 +1,32 @@
+
+//  Example 1 using visual debug
+//  Full primer is found in doc/chplvis/chplvis.rst.
+//  Please read that documentation for better understand chplvis.
```

Comment:
"for better understand" -> "to better understand" or "for a better understanding"